bundles/org.eclipse.osgi/container/src/org/eclipse/osgi/storage/FrameworkExtensionInstaller.java

```diff
@@ -80,8 +80,6 @@ private static Method findMethod(Class<?> clazz, String name, Class<?>[] args, C
 				result.setAccessible(true);
 			}
 			return result;
-		} catch (SecurityException e) {
-			// if we do not have the permissions then we will not find the method
```

Comment:
Why did you remove this?  I believe this will result in blowing up `<clinit>` if the framework does not have permission to reflect.

Reply:
Nevermind.  SecurityException is a RuntimeException!